dbt/models/mart/dim_city.sql

```diff
@@ -0,0 +1,52 @@
+{{ config(materialized='table') }}
+
+WITH pop_cities AS (
+
+    select * from {{ ref('stg_population') }}
+),
+
+cost_cities AS (
+
+    select * from {{ ref('stg_costs') }}
+
+),
+no_pop_dupes AS (
+
+    SELECT 
+        *
+    FROM pop_cities
+    INNER JOIN 
+            (
+            SELECT 
+                standard_city
+            FROM pop_cities
+            GROUP BY standard_city
+            HAVING count(*) = 1
+            )
+    USING (standard_city)
+),
+
+no_city_dupes AS (
+    SELECT 
+        *
+    FROM cost_cities
+    INNER JOIN 
+            (
+            SELECT 
+                standard_city
+            FROM cost_cities
+            GROUP BY standard_city
+            HAVING count(*) = 1
+            )
+    USING (standard_city)
+),
+
+cities AS (
+    
+    SELECT *
```

Comment:
one note about `*'s`

I tend to avoid them whenever possible. Lets say you make a column name change far upstream. Its nice to be find-replace throughout your code base to make sure you know and understand where that column is used and how it is updated. By having a `*` you aren't explicitly calling everything out. 

What do you think about adding all of the columns names here?

Reply:
(Im not concerned about them in the top-level CTE, but might be worth changing in the tables in between).